osbenchmark/benchmark.py

```diff
@@ -902,6 +912,7 @@ def dispatch_sub_command(arg_parser, args, cfg):
             generate(cfg)
         elif sub_command == "create-workload":
             cfg.add(config.Scope.applicationOverride, "generator", "indices", args.indices)
+            cfg.add(config.Scope.applicationOverride, "generator", "total_docs", args.total_docs)
```

Comment:
This could be changed to:
```
args.number_of_docs if len(args.number_of_docs) > 0 else None
```
but even that may not be necessary since an empty list matches a `False` predicate.